Templates/CRMs/Magento/Magento Products to Salesforce/long-description.md

```diff
@@ -0,0 +1,21 @@
+This template inserts Magento products and their corresponding prices to Salesforce.
+Using this template, you can insert your new Magento products to Salesforce.
+
+![Template](assets/Adobe_Commerce___Magento_Product_to_Salesforce.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The Salesforce user should have access to client ID, secret and security token.
+- Price book for Magento is initialized in Salesforce. It is labelled as "Magento Price Book" in this template.
+
+# Implementation and Usage Notes
+
+This template uses Salesforce queries to identify a product and the price book it's connected to. The price book, in which the product price is inserted, can be changed by altering the name in the price book task parameters.
```

Comment:
Should this template handle price updates as well? As a template user I would probably see this as a major flaw, if those prices are only moved to Salesforce once and then would never get updated. Is this a hard thing to do?

Reply:
I think it could be implemented after retrieving Magento price book information: We try to get existing price book entry information that matches with the product code. If no lines are returned, we insert the product as a new price book entry. If an entry is returned, we update the data with new price information that's found in Magento.

Do you want me to give it a go?